real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/LoggingPanel.java

```diff
@@ -79,6 +85,22 @@ public void setFileName(String fileName) {
 	}
 
 	/**
+	 * safe returns the speed
+	 * @return the speed, or 1 if the text in the field can't be serialized
+	 */
+	public double getPlaybackSpeed() {
+		String speedStr = playbackSpeed.getText();
+		if (speedStr.equals("")) return 1;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Position literals first in String comparisons](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2041124223/issues/source?bid=2938665&fileBranchId=3128705#l93)